spring-security-openid/pom.xml

```diff
@@ -21,7 +21,7 @@
 
     <properties>
         <modules.root.dir>${basedir}/../..</modules.root.dir>
-        <openid.version>5.0.5.RELEASE</openid.version>
+        <openid.version>4.2.4.RELEASE</openid.version>
```

Comment:
Looks like a lower version. I won't pretend to understand what's going on here, but just wanted to confirm you're changing to a lower version on purpose.

Reply:
Yes, this is an intentional change. This is changed in Aug towards the beginning 2-3 weeks of the projects - Spring 3.1 is used orignially, we planned to upgrade to Spring 5, but since Spring 5 doesnt support JDO and Spring documentation recommends to use Spring 4 ( supported till 2020 end) if we need JDO. We are using JDO in Motech datanucleus. 
 https://github.com/spring-projects/spring-framework/wiki/Upgrading-to-Spring-Framework-5.x